src/Raven.Server/Documents/TimeSeries/TimeSeriesRollups.cs

```diff
@@ -197,8 +197,9 @@ internal void PrepareRollups(DocumentsOperationContext context, DateTime current
                 if (rollUpTime > currentTicks)
                     return;
 
-                DocumentsStorage.TableValueToSlice(context, (int)RollupColumns.Key, ref item.Result.Reader,out var key);
+                DocumentsStorage.TableValueToSlice(context, (int)RollupColumns.Key, ref item.Result.Reader, out var key);
                 SplitKey(key, out var docId, out var name);
+                name = context.DocumentDatabase.DocumentsStorage.TimeSeriesStorage.GetOriginalName(context, docId, name);
```

Comment:
let's have it in try/catch and skip on error

Reply:
https://github.com/ravendb/ravendb/pull/10903